### PR TITLE
feat(aip_xx1_description): added cameras 6 and 7

### PR DESCRIPTION
**Pull request by knzo25**
Added the ids for cameras 6 and 7 as part of the c1 cameras installation.
The extrinsics corresponds to the ones taken on the 10/25

Signed-off-by: Kenzo Lobos-Tsunekawa <kenzo.lobos@tier4.jp>

### CODE REVIEW

**Comment by miursh** [comment]
@knzo25 
I think we start using calibration params in [aip_launcher](https://github.com/tier4/aip_launcher/blob/tier4/universe/aip_xx1_description/config/sensor_kit_calibration.yaml) instead of  [aip_description.iv](https://github.com/tier4/aip_description.iv/blob/main/aip_xx1_description/config/sensor_kit_calibration.yaml).
Maybe aip_description.iv is no longer used in the latest autoware.universe. 
So, could you make another PR to aip_launcher?

File: aip_xx1_description/urdf/sensor_kit.xacro [hunk 1]
```diff
@@ -197,6 +197,36 @@
       height="400"
       fov="1.3"
     />
+    <xacro:monocular_camera_macro
+      name="camera6/camera"
+      parent="sensor_kit_base_link"
+      namespace=""
+      x="${calibration['sensor_kit_base_link']['camera6/camera_link']['x']}"
+      y="${calibration['sensor_kit_base_link']['camera6/camera_link']['y']}"
+      z="${calibration['sensor_kit_base_link']['camera7/camera_link']['z']}"
```
Comment (Mingyu1991) [inline aip_xx1_description/urdf/sensor_kit.xacro:206]: camera7/camera_link -> camera6/camera_link?

**Comment by knzo25** [comment]
@miursh 
Sorry, I completely missed the last comment as I was out at the time. I will send the PR now

File: aip_xx1_description/urdf/sensor_kit.xacro (hunk 1 above)
Comment (knzo25) [reply aip_xx1_description/urdf/sensor_kit.xacro:206]: I will address it in the new repo

**Comment by knzo25** [comment]
@miursh 
I sent the PR in 
https://github.com/tier4/aip_launcher/pull/102

:bow:

**Pull request closed without merging**